### PR TITLE
Update state handle session management diagram

### PR DESCRIPTION
**Pull request by haishengwu-okta**
## Description:

- update state token session management diagram due to recent changes

## PR Checklist

- [ ] Have you verified the basic functionality for this change?

- [ ] Added unit tests?

- [ ] Added e2e tests

- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?

### Issue:

- https://oktainc.atlassian.net/browse/OKTA-412004

### CODE REVIEW

File: src/v2/models/AppState.js [hunk 1]
```diff
@@ -26,8 +26,8 @@ import { _ } from '../mixins/mixins';
 export default Model.extend({
 
   local: {
-    introspectSuccess: 'object', // only set during introspection
-    introspectError: 'object', // only set during introspection
+    introspectSuccess: 'object', // only set during introspection; TODO: remove me
+    introspectError: 'object', // only set during introspection; TODO: remove me
```
Comment (sherizada-okta) [inline src/v2/models/AppState.js:30]: Are there more details for this TODO?

**Review by sherizada-okta — COMMENTED** [review]
Image looks good

File: src/v2/models/AppState.js (hunk 1 above)
Comment (haishengwu-okta) [reply src/v2/models/AppState.js:30]: Those are depredate props and can be removed.

File: docs/manage-stateHandle-in-sessionStorage.org [hunk 2]
```diff
@@ -2,6 +2,15 @@
 @startuml
 start
 
+note left
```
Comment (pradeepdewda-okta) [inline docs/manage-stateHandle-in-sessionStorage.org:5]: We should ask devex team to update this related to their `interactionCode` flow as well

**Review by pradeepdewda-okta — APPROVED** [review]

**Review by sherizada-okta — APPROVED** [review]

File: docs/manage-stateHandle-in-sessionStorage.org (hunk 2 above)
Comment (haishengwu-okta) [reply docs/manage-stateHandle-in-sessionStorage.org:5]: @aaronbrodersen-okta @shuowu ^^

Comment (pradeepdewda-okta) [reply docs/manage-stateHandle-in-sessionStorage.org:5]: @aarongranick-okta correct Aaron.

**Pull request closed without merging**